dts/bindings/wifi/nordic,nrf700x.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+# GPIO lines for controlling the nRF700x Wi-Fi chip.
+include: base.yaml
+
+properties:
+  iovdd-ctrl-gpios:
+    type: phandle-array
+    required: true
+    description: |
+            GPIO of the SOC controlling IO VDD Control pin of the nRF700x
```

Comment:
These can be on the same line, if not, the indentation should be two spaces rather than 8 :)